diagnostic.md

```diff
@@ -94,4 +94,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+exercise, ? , weeks
```

Comment:
teamMember['name'] refers to the object { given: 'Jason', surname: 'Weeks' }.  This is what is known as a "nested object" since it's an object within another object.  Remember that strings need to be surrounded by single or double quotation marks and that case matters.